src/app/middleware.ts

```diff
@@ -59,13 +59,14 @@ startListening({
         : `fetching caps from lsp`;
       listenerApi.dispatch(setError(message));
     }
-
+    // do we need this?
```

Comment:
yes, for when the lsp isn't responding the and the `skip` option is set to true, the condition will be `skipped`